utils/table_printer.go

```diff
@@ -132,6 +111,91 @@ func (t *ttyTablePrinter) Render() error {
 	return nil
 }
 
+func (t *ttyTablePrinter) calculateColumnWidths(delimSize int) []int {
+	numCols := len(t.rows[0])
+	allColWidths := make([][]int, numCols)
+	for _, row := range t.rows {
+		for col, field := range row {
+			allColWidths[col] = append(allColWidths[col], field.DisplayWidth())
+		}
+	}
+
+	// calculate max & median content width per column
+	maxColWidths := make([]int, numCols)
+	// medianColWidth := make([]int, numCols)
```

Comment:
Looks like the `medianColWidth` is no longer used, can we delete the related comments?

Reply:
Good spot; I wanted to make use of median measurements to make weighted elastic resizing based on the contents of each column, but haven't gotten it right yet. Still, I would like to keep this around for now in case I have an epiphany later.